pkg/email/help_cant_find_something.go

```diff
@@ -0,0 +1,35 @@
+package email
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+)
+
+// SendCantFindSomethingEmailInput contains the data submitted by the user to the "can't find find
+// what you're looking for" help form
+type SendCantFindSomethingEmailInput struct {
+	Name  string
+	Email string
+	Body  string
+}
+
+// SendCantFindSomethingEmail sends an email to the EASI team containing a user's request for help
+func (c Client) SendCantFindSomethingEmail(ctx context.Context, input SendCantFindSomethingEmailInput) error {
+	subject := "Feedback for EASi System"
```

Comment:
Similar note to above, but I will need to reach out to Zoë to figure out a more apt title for this.